sql/parser/parse_test.go

```diff
@@ -84,6 +84,8 @@ func TestParse(t *testing.T) {
 		{`DROP TABLE a.b`},
 		{`DROP TABLE a, b`},
 		{`DROP TABLE IF EXISTS a`},
+		{`DROP INDEX a.b@c`},
```

Comment:
The `@` syntax feels unfortunate to me here. I'd rather see `a.b.c`. Nothing to be done in this PR, just talking out loud.